src/components/local/transaction/PrivateTradesB.js

```diff
@@ -37,7 +37,7 @@ export default class PrivateTradesB extends React.Component{
                             Header: TRADES.SIDE_TEXT,
                             id: TRADES.SIDE_TEXT,
                             class: "headerW",
-                            accessor: "side"
+                            accessor: d=>d.side === "BUY" && TRADES.BUY || d.side === "SELL" && TRADES.SELL
```

Comment:
please convert strings to lowercase and match as the incoming string can be modified by future developers and them may end up sending "buy" instead of "BUY"